bundles/extension.rst

```diff
@@ -97,7 +97,7 @@ For instance, assume you have a file called ``services.xml`` in the
     {
         $loader = new XmlFileLoader(
             $container,
-            new FileLocator(__DIR__.'/../Resources/config')
+            new FileLocator(__DIR__.'/../config')
```

Comment:
Do you pretend to upgrade the directory structure? In that case, the sentences above should be updated too.

Reply:
Oh, you're right. Yes, let's use the new bundle structure everywhere in the 6.1 docs :)

Reply:
Updated, also the path here is missing one '/../' as the extension class in this case is placed in `src/DependencyInjection/`